introduction.md

```diff
@@ -1,2 +1,5 @@
 Hello I am Clare. My favorite Code School path is HTML & CSS.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?